worker/src/main/java/org/apache/celeborn/service/deploy/worker/storage/PartitionDataWriter.java

```diff
@@ -153,6 +156,10 @@ public PartitionDataWriter(
       this.mapIdBitMap = new RoaringBitmap();
     }
     takeBuffer();
+    CongestionController congestionController = CongestionController.instance();
+    if (!isMemoryShuffleFile.get() && congestionController != null) {
+      userBufferInfo = congestionController.getUserBuffer(getDiskFileInfo().getUserIdentifier());
```

Comment:
IMO the CongestionController should control all file write operations including memory file ?

Reply:
I keep  the original code logic unchanged. In the original logic, CongestionController does not manage memory files. Maybe I can correct this logic in the next PR?

Reply:
> I keep the original code logic unchanged. In the original logic, CongestionController does not manage memory files. Maybe I can correct this logic in the next PR?

LGTM